pages/about/monthly-calls.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Past monthly calls
+permalink: /about/monthly-calls/
+redirect_from:
+- /documentation/video-tutorials/
+- /about/monthly-call/
+layout: styleguide
+category: About
+lead: At USWDS monthly calls, we share important updates and tips about how to use the Design System.
+subnav:
+- text: Latest updates
+  href: '#changelog'
+changelog:
+  key: about-monthly-calls
+
+---
+
+Register for [upcoming calls on Digital.gov](https://digital.gov/events/).
+
+## USWDS monthly call playlist
```

Comment:
```suggestion
## USWDS playlist
```
Recommend keeping this part more general in order to avoid confusion if another type of video is unexpectedly posted on the YouTube channel.